test/utils/audit.go

```diff
@@ -121,6 +121,7 @@ func CheckAuditList(el auditinternal.EventList, expected []AuditEvent) (missing
 	expectations := newAuditEventTracker(expected)
 
 	for _, e := range el.Items {
+		e := e
```

Comment:
hm, we pass e by ref on line below. And this may result in bad behavior if `testEventFromInternal` will run any goroutine or will keep the ref for later use.

Reply:
/hold

This may lead to an issue, https://github.com/kubernetes/kubernetes/pull/106371, is the change forced by linter?

Reply:
that issue fixed exactly this problem, we have to capture the variable or the reference will only use the last value, or am I getting wrong your concern?

Reply:
Oh, I read the PR backwards. Adding it makes a lot of sense. Sorry for the noise